Without pytest asyncio auto mode, tests fail with CollectionWasNotInitialized

Thanks for the repo, it was very helpful getting asyncio and pytest to work nicely.
Just one thing i discovered, you might need to add 

pytest.ini
```
[pytest]
asyncio_mode=auto
```

Otherwise tests fail with errors:
```
pytest
======================================================================== short test summary info =========================================================================
FAILED tests/routes/test_auth.py::test_not_authorized - AttributeError: 'async_generator' object has no attribute 'get'
FAILED tests/routes/test_auth.py::test_refresh - beanie.exceptions.CollectionWasNotInitialized
FAILED tests/routes/test_user.py::test_user_get - beanie.exceptions.CollectionWasNotInitialized
FAILED tests/routes/test_user.py::test_user_update - beanie.exceptions.CollectionWasNotInitialized
FAILED tests/routes/test_user.py::test_user_delete - beanie.exceptions.CollectionWasNotInitialized
```

--- a/pytest.ini
+++ b/pytest.ini
@@ -1,0 +1,2 @@
+[pytest]
+asyncio_mode=auto

